pre-reg-113-115-application-default.properties

```diff
@@ -0,0 +1,86 @@
+#helm chart properties
+#db.source.dbuser.password
+#db.destination.dbuser.password
+#server.port
+#mosip.base.url
+#mosip.prereg.client.secret
+#encryptBaseUrl
+#decryptBaseUrl
+#source.object.store.secretkey
+#destination.object.store.secretkey
+#isNewDatabase=true
+
+#Database config
+mosip.source.database.hostname=postgres-postgresql.postgres
+mosip.source.database.port=5432
+source.db.name=mosip_prereg_qa3
+# primary database connection
+datasource.primary.jdbcUrl=jdbc:postgresql://${mosip.source.database.hostname}:${mosip.source.database.port}/${source.db.name}
+datasource.primary.username=postgres
+datasource.primary.password=${db.source.dbuser.password}
+datasource.primary.driver-class-name=org.postgresql.Driver
+
+mosip.destination.database.hostname=postgres-postgresql.postgres
+mosip.destination.database.port=5432
+destination.db.name=mosip_prereg_qa_upgrade
+# secondary database connection
+datasource.secondary.jdbcUrl=jdbc:postgresql://${mosip.destination.database.hostname}:${mosip.destination.database.port}/${destination.db.name}
+datasource.secondary.username=postgres
+datasource.secondary.password=${db.destination.dbuser.password}
+datasource.secondary.driver-class-name=org.postgresql.Driver
+
+## Hibernate Properties
+# The SQL dialect makes Hibernate generate better SQL for the chosen database
+spring.jpa.properties.hibernate.dialect = org.hibernate.dialect.PostgreSQLDialect
+
+spring.main.web-application-type=none
+
+server.port=8081
+
+mosip.base.url=https://qa3.mosip.net
+cryptoResource.url=${mosip.base.url}/v1/keymanager
+
+appId=prereg
+clientId=mosip-prereg-client
+secretKey=${mosip.prereg.client.secret}
+
+decryptAppId=REGISTRATION
+decryptReferenceId=
+encryptAppId=PRE_REGISTRATION
+encryptReferenceId=INDIVIDUAL
+
+decryptBaseUrl=${mosip.base.url}
+encryptBaseUrl=https://qa-upgrade.mosip.net
+
+# Hibernate ddl auto (create, create-drop, validate, update)
+spring.jpa.hibernate.ddl-auto = update
+
+spring.main.allow-circular-references=true
+spring.cloud.compatibility-verifier.enabled=false
+
+mosip.kernel.objectstore.account-name=prereg
+
+object.store.s3.use.account.as.bucketname=true
+
+logging.level.com.amazonaws=debug
+
+# source object store server properties
+object.store.s3.url=http://127.0.0.1:9000
+object.store.s3.accesskey=minioadmin
+object.store.s3.secretkey=${source.object.store.secretkey}
+
+object.store.s3.region=
+object.store.s3.readlimit=10000000
+object.store.connection.max.retry=2
+
+# destination object store server properties
+destinationObjectStore.s3.url=http://127.0.0.1:9002
+destinationObjectStore.s3.access-key=minioadmin
+destinationObjectStore.s3.secret-key=${destination.object.store.secretkey}
+
+management.endpoints.web.exposure.include=refresh
+
+# set true if you want to set up new database and object store
+isNewDatabase=true
+
+spring.main.allow-bean-definition-overriding=true
```

Comment:
Can we keep this in internal properties file ?

Reply:
Done.